tests/test_options_flow.py

```
@@ -55,13 +55,13 @@
 ]
 
 
-async def test_options_flow_navigate_all_screens(
+async def test_options_flow_navigate_all_screens(  # noqa: PLR0913
```

Ruff (0.14.7)

Unused `noqa` directive (non-enabled: `PLR0913`)

Remove unused `noqa` directive

(RUF100)